README.md

```diff
@@ -6,31 +6,51 @@ and like MM is shortcut, this container is a simillar shortcut that saves time.
 :)
 
 ## Run it in few easy step.
-1. git clone this code to your docker machine.
-2. copy your cert in `certs/cert.pem` and `certs/privkey.pem`
-2. configure your app side configs
-   e.g. Set TURN server and credential in `configs/app/config.js`
-3. configure your server side configs
-   e.g. Set TURN server and credential in `configs/server/config.js`
-4. replace logo/logo.svg with your company logo svg.
+1. Git clone this code to your docker machine.
+2. Copy your cert in `certs/cert.pem` and `certs/privkey.pem`
+    1. In case you need to generate a new cert and private key, you can use (note -nodes flag, which allows to generate unencrypted private key)
+```
+        $ openssl req -x509 -newkey rsa:4096 -keyout privkey.pem -out cert.pem -days 365 -nodes
+```
+3. **Recomended**: set TURN server and credential in `configs/app/config.js`
+    1. In case you are using coturn, you can generate a user and key with
+```
+        $ turnadmin -k -u <username> -p <password>
+```
+Placeholder looks like 
+```
+	turnServers     : [
+		{
+			urls : [
+				'turn:example.com:443?transport=tcp'
+			],
+			username   : 'example',
+			credential : 'example'
+		}
+```
+You would need to replace example.com by your IP or domain, add the username previously used `<username>` and credential is the code generated by the above mentioned command.
+
+4. **Optional:** Change other stuff in config:
+    1. **Optional:** replace logo/logo.svg with your company logo svg.
+    2. **Optional:** sort audio/video codecs according to preference.
 
 ## Run:
 There is two ways
-1. simple use `docker run` command
+1. Simple use `docker run` command
 ```
-./run.sh
+$ sudo ./run.sh
 ```
 
-2. or with `docker-compose` 
+2. Or with `docker-compose` 
 / [install docker compose](https://docs.docker.com/compose/install/) /
 ```
-  docker-compose up --detach
+  $ sudo docker-compose up --detach
 ```
 ## Rebuild
 
 If you change app-config.js or or something in .env then you have to rebuild the image.
 ```
-  docker-compose up --build --detach
+  $ sudo docker-compose up --build --detach
```

Comment:
Depends of your settings user could be part of docker group  and then sudo not needed